### DISCUSSION TITLE
Tailwind Integration

### DISCUSSION THREAD

**Discussion by LubosSremanak** [primefaces/community#638] [PrimeNG] [answered]
When is the integration of Tailwind planned? Similar to PrimeReact and PrimeVue?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We'll implement passthrough, unstyled and tailwind in the Q1 2024

**Comment by dkbhadeshiya** [comment]
Hi Team, 

Can you provide updated timeline for Tailwind?

**Reply by raboud** [reply]
It is now Q3  2024 ANY update
